sklearn/linear_model/_stochastic_gradient.py

```diff
@@ -1606,3 +1618,442 @@ def _more_tags(self):
                 'zero sample_weight is not equivalent to removing samples',
             }
         }
+
+
+class SGDOneClassSVM(BaseSGD, OutlierMixin):
+    """Solves linear One-Class SVM using Stochastic Gradient Descent.
+
+    This implementation is meant to be used with a kernel approximation
+    technique to obtain results similar to `sklearn.svm.OneClassSVM` which uses
```

Comment:
Maybe mention more explicitly:

 ... to be used with in a pipeline with a kernel approximation technique such as `sklearn.kernel_approximation.Nystroem` to obtain results similar to ...